autotest/gcore/basic_test.py

```diff
@@ -40,11 +40,13 @@
 # Nothing exciting here. Just trying to open non existing files,
 # or empty names, or files that are not valid datasets...
 
+non_existing_error_msg = 'No such file or directory'
```

Comment:
The autotest also run on Windows and Mac.
For Windows, there's an AppVeyor config in this branch : https://github.com/rouault/gdal_coverage/blob/trunk_vc12_full/appveyor.yml  . It could/should probably be merged into main repo. For now, master is merged regularly in this branch.
For Mac, this is https://github.com/rouault/gdal_coverage/blob/trunk_travis_macosx/.travis.yml . Could probably be unified into the main travis.yml with some matrix logic.
With the current setup, I guess you could probably fork that repo and merge your branch into the 2 above branches, and create PR to trigger the builds ?